cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -21,17 +21,22 @@ def install():
     files_dir = os.path.join(hookenv.charm_dir(), 'files')
     archive = os.path.join(files_dir, 'kubernetes.tar.gz')
     command = 'tar -xvzf {0} -C {1}'.format(archive, files_dir)
+    print(command)
```

Comment:
i know this seems like pedantry but I wonder if we shouldn't pass these through the log() method instead of directly print()'ing.